login.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```

Comment:
❗️ 수정요청
타이틀은 유저 입장에서 해당 페이지를 파악하는 중요한 단서이므로 자세히 작성하시는 것을 추천드려요~

```suggestion
    <title>판다마켓 - 로그인</title>
```